.github/workflows/build.yml

```
@@ -0,0 +1,242 @@
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    outputs:
+    # Make the version accessible to other jobs
```

YAMLlint (1.38.0)

[warning] comment not indented like content

(comments-indentation)